src/riak_core_bucket.erl

```diff
@@ -192,14 +197,61 @@ bucket_nval_map(Ring) ->
 default_object_nval() ->
     riak_core_bucket:n_val(riak_core_bucket_props:defaults()).
 
+%% @private
+%% @doc For testing/debugging
+-spec all_n() -> [pos_integer(),...].
+all_n() ->
```

Comment:
I just realized I am opposed to this gratuitous function.  I will just call riak_core_ring_manager:get_my_ring() in the EQC test code.